torchbenchmark/util/framework/detectron2/__init__.py

```diff
@@ -55,6 +55,7 @@ def remove_tools_directory():
 
 def install_detectron2(model_name, model_dir):
     s3_utils.checkout_s3_data("INPUT_TARBALLS", "coco128.tar.gz", decompress=True)
```

Comment:
I think we could remove this? I remember detectron2 models only requires coco2017-minimal, not coco128 - I don't remember the details though.

Reply:
I grepped for coco128 and couldn't find anything other than instances of `checkout_s3_data`, so I think you're right - will do that.